modules/nextflow/src/main/groovy/nextflow/cli/CmdRun.groovy

```diff
@@ -310,7 +314,19 @@ class CmdRun extends CmdBase implements HubOptions {
 
         checkRunName()
 
-        log.info "N E X T F L O W  ~  version ${Const.APP_VER}"
+        if( launcher.options.ansiLog ){
+            log.debug "N E X T F L O W  ~  version ${Const.APP_VER}"
+
+            def fmt = ansi()
+            fmt = fmt.a("\n")
+            fmt = fmt.a(" 🚀  ")
```

Comment:
Not I'm ready for this 😆

Reply:
You have a favourite emoji you'd like to use instead? 👾 , ✨ or maybe 🙈 ? 😆

But yeah we can of course remove it. Looks pretty awesome with the proper Nextflow green background colour now, so not sure that the emoji really adds much!

Reply:
```suggestion
```